loan/README.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Loan
+category: Behavioral
+language: en
+tag:
+- Financial Management
+---
+
+## Intent
+Loan pattern is commonly used in financial system design. It helps ensure that loan objects are created and managed correctly, allowing for accurate calculation of interest, repayments, and other crucial financial metrics, without leading to data inconsistencies or inaccuracies.
+
```

Comment:
Please add "Explanation" section according to https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template